src/Compilers/CSharp/Portable/Symbols/Source/SourceLocalSymbol.cs

```diff
@@ -84,6 +84,22 @@ internal override SyntaxNode ScopeDesignatorOpt
             get { return _scopeBinder.ScopeDesignator; }
         }
 
+        internal override uint RefEscapeScope
```

Comment:
Consider using expression body for these properties.

Reply:
Will include in the next change. (do not want to rerun tests on this PR, since the bug is blocking)